### ISSUE TITLE
Will you accept a PR to lower the required PHP version to ^7.1?

### ISSUE THREAD

**Issue by willemstuursma** [mediact/dependency-guard#1]
Hi, Willem from Mollie here.

We saw this a few days ago on /r/php and would like to apply it to our applications. We've still got some on PHP 7.1, so hence my question: will you accept a PR to lower the required PHP version to ^7.1?

**Comment by johmanx10** [comment]
Absolutely. I've looked into it and we definitely want to broaden to at least `7.1`, since `iterable` was only used a couple of times as a means to speed up development, but never a goal. I do think this will cause the API to break, so we will release a version `2.0.0` after your pull request is approved.

We very much welcome your input and support!

**Comment by johmanx10** [comment]
When you open a pull request, please assign the following reviewers:

- Ashoka de Wit @sjokki 
- Jan-Marten de Boer @johmanx10

**Comment by willemstuursma** [comment]
`iterable` works in PHP 7.1 too, so a minor version bump is probably OK.
